frontend/src/Commons/Forms/InputModules/FieldsAndSoil/FieldsAndSoil.tsx

```diff
@@ -1,60 +1,57 @@
 /**
- *
- * THIS IS A SKELETON!
- * THIS IS STILL TO BE IMPLEMENTED!
- *
- * @summary Farm Information Input Module
+ * @summary Field and Soil Input Module
  * @description A module with input fields to be used in a Form Module
  * @author @GDamaso
```

Comment:
Don't forget to author this, I just built a skeleton, you did 99% of this implementation.

Reply:
Was poking around. I'll change it when's done. I was estimating the work to be done before I summarize everything. 😂